autosubmit/platforms/paramiko_platform.py

```diff
@@ -245,45 +317,18 @@ def interactive_auth_handler(self, title, instructions, prompt_list):
                     answers.append(twofactor_nonpush)
         return tuple(answers)
 
-    def map_user_config_file(self, as_conf) -> None:
```

Comment:
Had to mock the SSH Client, and the SSH config, and the code was linked to this function, so I moved it up to a function (not using anything really from the object) and will check if we need some more tests.